Projects/Online_Cloth_Strore/aboutus.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>About us - primrose</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <style>

```

Comment:
If possible please don't use any style tag in the HTML File!
Please move all the style code in the separate CSS file it's very bad for readibility.
When someone opens your HTML File they doesn't expect to see styling there. They just want to see your markup so please move this style in separate file

Reply:
In response to your suggestion, I've made the necessary adjustments to the code. The styles have been moved from the HTML file to a separate CSS file (about.css). This change adheres to better coding standards and enhances the overall organization of the project.